### PR TITLE
Check the initial capacity of HashMap

### PR DESCRIPTION
**Pull request by yx9o**
Check the initial capacity of HashMap in ss, Keep the code style consistent.

### CODE REVIEW

File: shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/schema/builder/SchemaPrivilegeBuilder.java [hunk 1]
```diff
@@ -67,7 +67,7 @@
      */
     private static Map<ShardingSphereUser, String> convertSchemas(final String userSchemaMapping) {
         String[] userSchemaPair = userSchemaMapping.trim().split("=");
-        Map<ShardingSphereUser, String> result = new HashMap<>();
+        Map<ShardingSphereUser, String> result = new HashMap<>(1, 1);
```
Comment (tristaZero) [inline shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/schema/builder/SchemaPrivilegeBuilder.java:70]: Hi Is there any possibility that it's not 1?

**Review by tristaZero — CHANGES_REQUESTED** [review]
Hi @yx9o Sorry for my late response. ;(
But, you really take good care of these particulars, nice.

File: shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/schema/builder/SchemaPrivilegeBuilder.java (hunk 1 above)
Comment (yx9o) [reply shardingsphere-infra/shardingsphere-infra-authority/shardingsphere-infra-authority-common/src/main/java/org/apache/shardingsphere/authority/provider/schema/builder/SchemaPrivilegeBuilder.java:70]: Hi, I read the context-related code. This method only processes one userSchemaMapping at a time. Line 76 of the code will put only one element in the map and then execute return. In subsequent processes, only traversal of the map is involved and no element will be put anymore.

**Review by tristaZero — APPROVED** [review]
@yx9o Sorry I came back late.
I saw the context of `SchemaPrivilegeBuilder`, which looks it has some room to improve. But thanks for your tweak here. ;-)

**Comment by yx9o** [comment]
Hi, I think if there is only one `userSchemaMapping` at a time, the `convertSchemas` and `merge` methods can be merged, and the logic will be clearer.

**Comment by tristaZero** [comment]
@yx9o If you like, welcome the next PR to improve it. ;-)

**Comment by yx9o** [comment]
ok

**Pull request merged**